queries/t32/highlights.scm

```diff
@@ -76,12 +76,12 @@
 (
   (command_expression
     command: (identifier) @keyword.return)
-  (#match? @keyword.return "^[eE][nN][dD]([dD][oO])?$")
+  (#any-of? @keyword.return "end" "END" "enddo" "ENDDO")
 )
 (
   (command_expression
     command: (identifier) @keyword.return)
-  (#match? @keyword.return "^[rR][eE][tT][uU][rR][nN]$")
```

Comment:
Do languages like this allow for things like `rEtUrN` or not?

Reply:
Presumably. But would anyone write it like that?

Reply:
If the grammar allows it, it should be highlighted correctly.